oeps/processes/oep-0063-proc-toc-decision.rst

```diff
@@ -0,0 +1,112 @@
+.. _pep_based_template:
+
+.. Below is the display in the left sidebar on RTD. Please omit leading 0's
+
+OEP-63: TOC Decision
```

Comment:
This title still doesn't capture the gist of this proposal for me.  Something like "Getting TOC Support," though somewhat informal, is really what we're trying to specify I think.

Reply:
@e0d The lack of formality in "Getting TOC Support" is an issue imho - it is meant to be an important formal action, weakening the title weakens the action, and the expected response. 

What's the part that doesn't match the proposal in "TOC Decision"? The fact that it doesn't contain the part about the community requesting it from the TOC? Maybe "TOC Decision Request" would make this aspect clearer?

Reply:
I suppose I see two issues here:
1. I don't think TOC support should be limited to decisions, there should be avenues for support developing a plan or approach as well, when the scope of the change warrants it.
2. I think an active title would be convey the purpose of the document better.  For me this is still leading with the last resort, TOC breaks the tie because the project is stuck.

Reply:
@e0d Thanks for the review!

> I don't think TOC support should be limited to decisions, there should be avenues for support developing a plan or approach as well, when the scope of the change warrants it.

I see, it could make sense to use a word with a broader meaning. The difficulty is to find one that remains meaningful. How about "TOC Resolution Request"? "Resolution" is defined as _"a formal expression of the opinion or will of the official body"_, and makes it clear that there will be a concrete and well defined outcome, while being broader than just taking a decision.

> I think an active title would be convey the purpose of the document better. For me this is still leading with the last resort, TOC breaks the tie because the project is stuck.

I've made changes to the beginning of the document, to make it broader than this specific example of the TOC breaking a tie, and more about intervening on the topic. See if that's better, or if you still see elements that are too narrow? https://github.com/openedx/open-edx-proposals/pull/484/commits/db0eadd8ef0a86393a5a7fa06a45488a1edb713e#diff-e1f9259a794a90d6d39a6b1fd5cc2b91258a1138f34516ea6db0f9a1ba99c0c3L39-L48